### DISCUSSION TITLE
Get vertex data out of Polygons in Python

### DISCUSSION THREAD

**Discussion by johnnybigert** [elalish/manifold#439] [Q&A]
Probably a stupid question, but how do I retrieve the vertex data from a `Polygons` object in Python? I've tried `list(polygons)`, `polygons[0]` and similar.

I see that there's a wrapper object `PolygonsWrapper` (which is a smart pointer to the vector of vector of float pairs I'm looking for), but the Python bindings does not provide any kind of accessor to the data itself, only extrude and revolve.

**Comment by elalish** [comment]
@pca006132 I think you've used the Python side more than I have - do you know?
